mmdet/models/data_preprocessors/data_preprocessor.py

```diff
@@ -333,3 +334,85 @@ def forward(
                         value=self.seg_pad_value)
 
         return batch_inputs, batch_data_samples
+
+
+@MODELS.register_module()
+class MultiBranchDataPreprocessor(nn.Module):
```

Comment:
This module should inherits from the BaseDataPreprocessor to have compatible functionalities.